internal/parseutil/filereader_test.go

```diff
@@ -0,0 +1,48 @@
+package parseutil
+
+import (
+	"strings"
+	"testing"
+)
+
+func TestLineResolution(t *testing.T) {
+	fr := &FileReader{}
+
+	file1 := "file1"
+	file2 := "file2"
```

Comment:
I think these constants obscure the test rather than clarify. Let's just inline them.